docs/map_state/embeddings.md

```diff
@@ -7,18 +7,12 @@ projected representations.
 ```python
 from nomic import AtlasProject
 
-project = AtlasProject(name='My Project')
+map = AtlasProject(name='My Project').maps[0]
```

Comment:
Probably not the place for this, but most of my workflow has been creating a single map per project. What's the use case for multiple maps per single project?

Reply:
if you index the project multiple times (say on different fields), you will have a map per index so possibly multiple indices